v20.2/debug-and-error-logs.md

```diff
@@ -112,6 +112,23 @@ These logging flags are used with [`cockroach` commands](cockroach-commands.html
 
 {% include {{ page.version.version }}/misc/logging-flags.md %}
 
+## Redacted logs
+
+If you contact CockroachDB Support for troubleshooting help, you might be asked to run [`cockroach debug zip`](cockroach-debug-zip.html) and share the resulting file with the CockroachDB team. The log files created by `cockroach debug zip` may contain highly sensitive, unanonymized information, such as usernames, hashed passwords, and possibly your table's data.
```

Comment:
Suggest working in the phrase "personally identifiable information (PII)" since that is a standard term that will also be searched for

Reply:
Raphael recommended not using the term (PII) and saying "sensitive, unanonymized information" instead. I agree with you though -- IMO, PII is a standard term that should feature somewhere in the doc. cc @knz for approval.